src/Common/DirectMessaging/ErrorConstants.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * ErrorConstants holds a number of string constants and error codes
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <snielson@discoverandchange.com>
+ * @copyright Copyright (c) 2022 Stephen Nielson <snielson@discoverandchange.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Common\DirectMessaging;
+
+class ErrorConstants
+{
+    // Message that is sent back to the user (translated) if messaging is unavailable
+    const MESSAGING_DISABLED = "Direct messaging is currently unavailable.";
```

Comment:
if do something like this, then would also place the constant in a comment wrapped in xl() (this then will get picked up by the translation scripts when collecting new constants).
```
// xl("Direct messaging is currently unavailable.")
```